### DISCUSSION TITLE
How do I upgrade to 20210416?

### DISCUSSION THREAD

**Discussion by apastuszak** [rern/rAudio#311] [General]
I've been running ```pacman -Syu``` to do updates, but the last update to rune audio that I got was 20210330.

**Comment by EarsofTin** [comment]
Try settings > add-ons > updates.   Unless you mean update to 64bit 20210416 which requires writing a new image to the SD card.

**Reply by apastuszak** [reply]
Thank you!

**Reply by findnix** [reply]
Hi!
I guess `/srv/http/data/addons/r1` is missing.
check:
`ls -l /srv/http/data/addons/r1`
if no `r1`:
`echo "20210330" > /srv/http/data/addons/r1`
